### PR TITLE
Add Kotlin Wrapper

### PR DESCRIPTION
**Pull request by desiderantes**
This PR add a complete kotlin implementation, along with a basic test. In this case, this is restricted to kotlin over JVM, but it can be tweaked to work with Kotlin over JS if desired

### CODE REVIEW

File: dist/kotlin/src/test/kotlin/ValidatorTests.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import org.spekframework.spek2.style.specification.describe
+
+object ValidatorTests : Spek( {
+
+    describe("The validator") {
+        mapOf(
+            "valid@gmail.com" to true,
+            "valid@yahoo.com" to true,
+            "qwerty" to false,
+            "asdf@asdf@asdf" to false,
```
Comment (ashish8947) [inline dist/kotlin/src/test/kotlin/ValidatorTests.kt:12]: @desiderantes 
Please fix  "," from the last line it is causing an error

Comment (desiderantes) [reply dist/kotlin/src/test/kotlin/ValidatorTests.kt]: Fixed

**Comment by ashish8947** [comment]
@desiderantes Thanks for your contribution. Getting exceptions while running the test cases please fix the same 
For Example -
```cmd
ValidatorTests > Validates to true FAILED
    java.lang.ExceptionInInitializerError at ValidatorTests.kt:16
        Caused by: com.beust.klaxon.KlaxonException at ValidatorTests.kt:16

```

**Comment by desiderantes** [comment]
Branch updated, please re-review

**Comment by desiderantes** [comment]
No response, closing

**Pull request closed without merging**